recipe/meta.yaml

```diff
@@ -113,8 +120,9 @@ outputs:
       imports:
         - libmambapy
         - libmambapy.bindings
+        - libmambapy.solver
       commands:
-        - python -c "import libmambapy._version; assert libmambapy._version.__version__ == '{{ libmambapy_version }}'"
+        - python -c "import libmambapy.version; assert libmambapy.version.__version__ == '{{ libmambapy_version }}'"
```

Comment:
Could run their unit tests for the python module
https://github.com/mamba-org/mamba/tree/main/libmambapy/tests